src/test/java/com/indeed/jiraactions/ActionTest.java

```diff
@@ -45,6 +48,7 @@ public void initialize() {
         EasyMock.replay(config);
 
         actionFactory = new ActionFactory(userLookupService, new CustomFieldApiParser(userLookupService), config);
+        statusTimeFactory = new StatusTimeFactory();
```

Comment:
Nit: Seems like we could just instantiate this one as a class variable. Methods annotated with `@Before` get executed before each test.